log/logger.go

```diff
@@ -71,6 +78,11 @@ func (l *Logger) GetFilePath() string {
 	return l.path
 }
 
+// Get the HTTP client tracer for this logger if HTTP tracing was enabled
+func (l *Logger) GetHttpTracer() *httptrace.ClientTrace {
+	return l.tracer
```

Comment:
might as well just expose this field (i'd rename to HttpTracer) and conditionally populate it in NewLogger when options.EnableHttpTracing is true